templates/vue/src/components/NodeModal.vue

```diff
@@ -516,7 +555,7 @@ export default {
             type: "",
           }
           await this.addLink(newLink)
-          if (this.node.status !== "draft") {
+          if (this.node.status !== "draft" && this.node.status !== "submitted") {
```

Comment:
It's not easy to understand why these conditions are here. It would be great if you could create some calculated values that are named in a way to describe what you are checking for exactly or otherwise add a comment to say why this condition is being added.

Reply:
Clarified